src/scripts/browserDeviceProfile.js

```diff
@@ -188,6 +188,27 @@ function supportsVc1(videoTestElement) {
     return browser.tizen || browser.web0s || browser.edgeUwp || videoTestElement.canPlayType('video/mp4; codecs="vc-1"').replace(/no/, '');
 }
 
+function supportsHdr10(options) {
+    return options.supportsHdr10 ?? (false
```

Comment:
`false` here just to make future changes easier.

Reply:
ok I have no objections to that... just need to add comments to suppress the eslint error for these lines :+1: